docs/migration/3_0.md

```diff
@@ -391,6 +391,14 @@ Now it is:
 ### OccCmsComponentAdapter
 The OCC CMS component API in SAP Commerce Cloud before version 1905 was using a POST method. This has changed in 1905 (using GET going forward). Spartacus has supported both version from version 1.0 by using a `legacy` flag to distinguish this backend API behavior. With release 3.0 we maintain the support for the pre 1905 CMS component API, but the implementation has moved to a separate adapter. With that change, we're also dropping the `legacy` flag in the OCC configuration. 
 
+## CloseAccountModalComponent
```

Comment:
Didn't notice this before, but ### instead of ##

Reply:
same for the rest below

Reply:
Done